woodwork/tests/utils/test_read_file.py

```diff
@@ -113,6 +113,10 @@ def test_read_file(sample_df_pandas, tmpdir, filepath, exportfn, kwargs, pandas_
                       logical_types=kwargs.get('logical_types'),
                       semantic_tags=kwargs.get('semantic_tags'),
                       column_origins=kwargs.get('column_origins'))
+
+    if "to_csv":
+        df.ww.logical_types['signup_date'].datetime_format = None  # read_csv reads datetimes as strings and infers datetime during transform
```

Comment:
This if block will always get executed because "to_csv" is truthy. What are we trying to do here?

Reply:
that was supposed to be checking the import func!